src/scripts/import-open-api.ts

```diff
@@ -267,7 +272,7 @@ export const generateRestfulComponent = (
   const needAResponseComponent = responseTypes.includes("{");
 
   // We ignore last param of DELETE action, the last params should be the `id` and it's given after in restful-react
```

Comment:
What does this mean `the last params should be the `id` and it's given after in restful-react`?

Reply:
There's a _lot_ of English changes to come to this PR. My plan is to add a commit with updated docs/comments/etc.

IMO we can resolve this for now and then add docs at the end as a final touch.

Reply:
@TejasQ As long as we don't forget. This is not docs, this is a code comment. In my opinion, comments should be fixed with the code they comment on.

Reply:
> My plan is to add a commit with updated docs/**comments**/etc.

We won't forget. 😄